cypress/integration/models/developer/applicationinventory/application.ts

```diff
@@ -22,6 +22,8 @@ import {
     deleteAction,
     assessment,
     tagCount,
+    git,
+    subversion,
```

Comment:
These are not required.

Reply:
Thanks for noticing it. Done :+1: 
@sshveta Please review it again. Thanks in advance !